src/vma/dev/ring_simple.cpp

```diff
@@ -1278,7 +1278,6 @@ bool ring_simple::is_available_qp_wr(bool b_block)
 					if (ret <= 0) {
 						ring_logdbg("failed blocking on tx cq_mgr (errno=%d %m)", errno);
 						m_lock_ring_tx_buf_wait.unlock();
-						m_lock_ring_tx.lock();
```

Comment:
@rosenbaumalex I looked at this code long time to understand lock/unlock mechanism in this function. I found that ring_simple::mem_buf_tx_get() and ring_simple::is_available_qp_wr() implementations are very close but there is difference in lock/unlock that was reported by coverity. I guess that current code could be reasonable but I could not understand a reason. So let keep one as is, I will suppress issue using special tag and create issue to investigate.
Do you think it is a way to go?